Command/NotifyDeploymentCommand.php

```diff
@@ -25,8 +25,6 @@ class NotifyDeploymentCommand extends Command
     public const EXIT_UNAUTHORIZED = 2;
     public const EXIT_HTTP_ERROR = 3;
 
-    protected static $defaultName = 'newrelic:notify-deployment';
```

Comment:
removing a `protected` property is a BC break

Reply:
It's overriding existing property, so it's still there

Reply:
In Symfony 6.1, the `$defaultName` property is no longer supported. The recommended ways to set the command name are via [service tag (backwards compatible through Symfony 3.4)](https://symfony.com/doc/3.4/console/commands_as_services.html#lazy-loading) or a [new PHP 8.0 Attribute](https://symfony.com/doc/3.4/console/commands_as_services.html#lazy-loading).